cms_das_query/xsdb_query.py

```diff
@@ -18,19 +18,23 @@ def xsdb_query(df, out_file=None, attrs=["shower","mtrx_gen","cross_section","ac
     # Loop over dataset querying using the "process_name"
     data = []
     for process_name in df["dataset"]:
-        query = {"process_name": process_name}
+        query = {"DAS": process_name}
```

Comment:
Any reason to change this name?

Reply:
It didn't seem to work otherwise.  When I changed it to `DAS` things worked, but if not it didn't return anything.  Perhaps I was querying the with the wrong string, but since I was running the script without specifying a dataset name (no `-d` option), the string passed here was just the first substring between backslashes in the dataset query for das (if that makes sense)

Reply:
For example:
```
$ python -c "from request_wrapper import RequestWrapper; RequestWrapper().simple_search({'request_name':'QCD_HT1000to1500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8'})
```
shows no output.  On the other using `DAS` I see:
```
$ python -c "from request_wrapper import RequestWrapper; RequestWrapper().simple_search({'DAS':'QCD_HT1000to1500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8'})" 
[{"status": "approved", "MCM": "SUS-RunIISummer16MiniAODv2-00078", "process_name": "QCD_HT1000to1500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8", "modifiedOn": "2017-12-06 21:02:06", "energy": "13", "fraction_negative_weight": "0.0", "equivalent_lumi": "0.8285", "comments": "Automatically computed", "shower": "Pythia8", "total_uncertainty": "1.166", "approvedBy": "perrozzi", "modifiedBy": "perrozzi", "createdBy": "cmsxsec", "DAS": "/QCD_HT1000to1500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8/RunIISummer16MiniAODv2-PUMoriond17_80X_mcRun2_asymptotic_2016_TrancheIV_v6-v1/MINIAODSIM", "cross_section": "1207.0", "mtrx_gen": "Madgraph", "_id": {"$oid": "5a280b8af20ea5766a5d4ce2"}, "createdOn": "2017-12-06 15:23:54", "accuracy": "LO"}]
```

Reply:
The first one seems to have `'request_name'` instead of `'process_name'`. Not sure why. If I do the search on the webpage then querying `process_name=QCD_HT1000to1500_TuneCUETP8M1_13TeV-madgraphMLM-pythia8` it seems to get the same result.

Need to double-check how using `DAS` to query will affect some searches such as for `DYJetsToLL_Pt-100To250_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8`, since it gives more than 1 result. Might need some logic on which to choose. Could use `DAS` or `process_name` as a backup for the other if it fails.

But for this PR, any idea why the command has `'request_name'` instead of `'process_name'`?

Reply:
Ah, you're right.  I think I messed that up!  I was just trying those commands inline and I think I must have made a mistake putting in the key.  
I think you're right then, `process_name` is probably better since you should only get one result I think?  I think the matching on xsdb is just done on whether the named field contains the string you query with (I'm trying to find the documentation I looked at that said this but I can't find it now).  So `DAS = QCD` will match every entry in the DB where the value for DAS contains the string `QCD`.

Reply:
Yeah, sorry for that.  I'll revert this change.

Reply:
No worries. Maybe we want multiple results and we can take the best calculation. I.e. rather than the value from genXsTool they have the calculation from NNLO. But for this PR we'll leave it as is